datafusion/substrait/tests/testdata/data.csv

```diff
@@ -1,3 +1,3 @@
 a,b,c,d,e,f
-1,2.0,2020-01-01,false,4294967296,'a'
-3,4.5,2020-01-01,true,2147483648,'b'
\ No newline at end of file
+1,2.0,2020-01-01,false,4294967295,'a'
```

Comment:
Why was this changed?

Reply:
The old value was 1 larger than what fits into uint64, so executing against this file failed.

I didn’t see any reason why the over-large value would be useful, so this brings it down into the range that actually fits into uint64

Reply:
I think you mean `Int32` 🤔 

```sql
> select 4294967295::int;
Arrow error: Cast error: Can't cast value 4294967295 to type Int32
> select 4294967295::bigint;
+-------------------+
| Int64(4294967295) |
+-------------------+
| 4294967295        |
+-------------------+
1 row(s) fetched.
Elapsed 0.002 seconds.
```

Reply:
Ah yes, I misremembered. It’s int32 as specified here https://github.com/apache/datafusion/blob/2aa57386c12e57d2fe865360f3eead358cc237f7/datafusion/substrait/tests/cases/roundtrip_logical_plan.rs#L1125. Maybe the original number at some point had some meaning to test the int32/int64 difference, but I don’t see what purpose it’s serve in the substrait roundtrip tests 😅